lib/chef/mixin/cron_validations.rb

```diff
@@ -0,0 +1,90 @@
+#
+# Copyright:: Copyright 2020, Chef Software Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+class Chef
+  module Mixin
+    # a collection of methods for validating cron times. Used in the various cron-like resources
+    module CronValidations
```

Comment:
This is called `Chef::Mixin::CronValidations` but it's not usable as a mixin...

Reply:
The word mixin wildly abused in chef/chef

Reply:
Renamed